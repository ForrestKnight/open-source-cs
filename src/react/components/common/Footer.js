/* Packages */
import React from 'react';


/* Component */
export class Footer extends React.Component {
    render() {
        return (
            <footer id="Footer">

            </footer>
        );
    };
};


export default Footer;