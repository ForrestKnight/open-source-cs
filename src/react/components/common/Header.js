/* Packages */
import React from 'react';


/* Components */
import Navigation from '../ui/Navigation';


/* Component */
export class Header extends React.Component {
    render() {
        return (
            <header id="Header">
                <Navigation />
            </header>
        );
    };
};


export default Header;