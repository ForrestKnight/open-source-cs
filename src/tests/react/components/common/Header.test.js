/* Packages */
import React from 'react';
import {shallow} from 'enzyme';


/* Component */
import {Header} from '../../../../react/components/common/Header';


/* Test Config */
let component;
beforeEach(() => {
    component = shallow(<Header />);
});


/* Tests */
test('Render Header component', () => {
    expect(component).toMatchSnapshot();
});