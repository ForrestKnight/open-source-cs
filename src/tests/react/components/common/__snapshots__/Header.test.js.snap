// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Header component 1`] = `
<header
  id="Header"
>
  <Navigation />
</header>
`;
